core/src/i18n/en.yaml

```diff
@@ -2,18 +2,18 @@ addons:
   addons-status: Addons Status
   description: <0>Addons Configuration</0> defines Namespace-scoped addons that provide all information required to convert Helm charts into Service Classes in Service Catalog.
   errors:
-    cannot-create: Cannot create the Addons Configuration.
+    cannot-create: Cannot create the Addons Configuration
   headers:
     url: URL
   messages:
-    created: Addons Configuration {{name}} created.
+    created: Addons Configuration {{name}} created
   repository-urls: Repository URLs
   singular_name: Addons Configuration
   title: Addons
   tooltips:
-    failed: One of the repositories failed.
-    pending: Addon is being processed.
-    ready: Addon configured.
+    failed: One of the repositories failed
```

Comment:
In case of errors, it's good to add information why an error occurred and what the user should do to fix it.